### PR TITLE
Bump version, oops!

### PR DESCRIPTION
**Pull request by Mstrodl**
Forgot to bump the version in my last PR(!)
This does that and adds it to the changelog as well as the previous nullsafety version.

### CODE REVIEW

**Comment by Mstrodl** [comment]
Adding @jathak here as well, should I keep the `-nullsafety` tag at the end of future versions?

**Comment by Awjin** [comment]
@Mstrodl We decided to [revert the null-safe changes](https://github.com/mbullington/node_preamble.dart/pull/24) for now. If you rebase this PR on master and bump the version to `1.4.13` we should be good to go.

**Comment by Mstrodl** [comment]
@Awjin All done, just need an approving review :p

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 1.4.14
+
+* Fixes detection on Electron with `nodeIntegration` disabled
+
+# 1.4.13-nullsafety
```
Comment (Awjin) [inline CHANGELOG.md]: You can remove this entry and instead mark the Electron entry as 1.4.13

**Review by Awjin — CHANGES_REQUESTED** [review]

**Comment by Mstrodl** [comment]
going to cc @mbullington because I believe we still need you to publish to pub.dev :)

**Review by Awjin — APPROVED** [review]

**Comment by mbullington** [comment]
@Mstrodl Taken care of, published. Thanks for helping maintain all, cheers!

**Pull request merged**